run_integration.sh

```diff
@@ -7,7 +7,8 @@ pushd $rootdir
 
 # Reset test harness
 rm -rf test-harness
-git clone --single-branch --branch master https://github.com/algorand/algorand-sdk-testing.git test-harness
+# TODO: Before merging, change branch back to master
+git clone --single-branch --branch feature/box-storage https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
TODO: Change this branch back to master after merging in the Algorand SDK Testing changes